app/javascript/src/hyacinth_v1/components/digital_objects/children/ChildStructureEditor.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Badge, Card } from 'react-bootstrap';
+import { startCase } from 'lodash';
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+
+const ChildStructureEditor = (props) => {
+  const {
+    digitalObjects, onRearrange,
+  } = props;
+
+  const [currentObjectList, setCurrentObjectList] = useState(digitalObjects);
+
+  const onDragEnd = (result) => {
+    const { destination, source } = result;
+
+    if (!destination) {
+      return;
+    }
+
+    // no changes
+    if (
+      destination.droppableId === source.droppableId
+      && destination.index === source.index
+    ) {
+      return;
+    }
+
+    const newList = Array.from(currentObjectList);
+    // insert dragged element to new location
+    newList.splice(destination.index, 0, newList.splice(source.index, 1)[0]);
+    setCurrentObjectList(newList);
+    onRearrange(newList);
+  };
+
+  return (
+    <>
+      <DragDropContext onDragEnd={onDragEnd}>
+
+        <Droppable droppableId="droppable-1">
+          {(DroppableProvided) => (
+            <div
+              ref={DroppableProvided.innerRef}
+              {...DroppableProvided.droppableProps}
+            >
+              {
+        currentObjectList.map((digitalObject, resultIndex) => (
+          <Draggable
+            draggableId={
+            digitalObject.id
+}
+            index={resultIndex}
+            key={digitalObject.id}
+          >
+            {(provided) => (
+              <Card
+                key={digitalObject.id}
+                className="mb-3"
+                {...provided.draggableProps}
+                {...provided.dragHandleProps}
+                ref={provided.innerRef}
+              >
+                <Card.Header>
+                  <Link
+                    to={`/digital_objects/${digitalObject.id}`}
+                  >
+                    {digitalObject.displayLabel}
+                  </Link>
+                </Card.Header>
+                <Card.Body>
+                  <ul className="list-unstyled small">
+                    <li>
+                      <strong>ID: </strong>
+                      {digitalObject.id}
+                    </li>
+                    {
+                  digitalObject.numberOfChildren > 0 && (
+                    <li>
+                      <strong>Children: </strong>
+                      {digitalObject.numberOfChildren}
+                    </li>
+                  )
+                }
+                  </ul>
+                  <Badge bg="secondary">{startCase(digitalObject.digitalObjectType)}</Badge>
+                </Card.Body>
+              </Card>
+            )}
+          </Draggable>
+
+        ))
+      }
+
+              {DroppableProvided.placeholder}
+            </div>
+
+          )}
+        </Droppable>
+      </DragDropContext>
+
+    </>
```

Comment:
```suggestion
    <DragDropContext onDragEnd={onDragEnd}>
      <Droppable droppableId="droppable-1">
        {(DroppableProvided) => (
          <div
            ref={DroppableProvided.innerRef}
            {...DroppableProvided.droppableProps}
          >
            {
              currentObjectList.map((digitalObject, resultIndex) => (
                <Draggable
                  draggableId={
                    digitalObject.id
                  }
                  index={resultIndex}
                  key={digitalObject.id}
                >
                  {(provided) => (
                    <Card
                      key={digitalObject.id}
                      className="mb-3"
                      {...provided.draggableProps}
                      {...provided.dragHandleProps}
                      ref={provided.innerRef}
                    >
                      <Card.Header>
                        <Link
                          to={`/digital_objects/${digitalObject.id}`}
                        >
                          {digitalObject.displayLabel}
                        </Link>
                      </Card.Header>
                      <Card.Body>
                        <ul className="list-unstyled small">
                          <li>
                            <strong>ID: </strong>
                            {digitalObject.id}
                          </li>
                          {
                            digitalObject.numberOfChildren > 0 && (
                              <li>
                                <strong>Children: </strong>
                                {digitalObject.numberOfChildren}
                              </li>
                            )
                          }
                        </ul>
                        <Badge bg="secondary">{startCase(digitalObject.digitalObjectType)}</Badge>
                      </Card.Body>
                    </Card>
                  )}
                </Draggable>

              ))
            }

            {DroppableProvided.placeholder}
          </div>

        )}
      </Droppable>
    </DragDropContext>
```
Since you have a single outer element, no need for the outer fragment tags.

Reply:
This change suggestion diff looks like a lot more than it actually is because I made some alignment fixes, but substantively the only real change is a deletion of the `<>` and `</>` fragment tags.